app/assets/src/components/views/compare/SamplesHeatmapView/SamplesHeatmapControls.jsx

```diff
@@ -24,6 +24,8 @@ import Slider from "~ui/controls/Slider";
 import SequentialLegendVis from "~/components/visualizations/legends/SequentialLegendVis.jsx";
 import ThresholdFilterTag from "~/components/common/ThresholdFilterTag";
 import FilterTag from "~ui/controls/FilterTag";
+import ColumnHeaderTooltip from "~/components/ui/containers/ColumnHeaderTooltip";
+import InfoSIcon from "~/components/ui/icons/InfoSIcon";
```

Comment:
Dont import it direclty like this. User the index in `ui/icons/index.js`.

Reply:
Done